### PR TITLE
Changed 'Prerequisite For' in course's additional information section…

### LINKED ISSUE #269: In CourseInfoBar, rename "Prerequisite For" to "Prerequisite for"
When you click on a course's name, it displays additional information. 
The "for" in "Prerequisite For:" should not be uppercase. Make it lowercase.

image

### PR DESCRIPTION
**Pull request by LawrenceBLu**
… to 'Prerequisite for'

## Summary
I changed the "For" in the course's additional information section to a lowercase "for"

image

## Test Plan
- Verified that the lowercase "for" shows up instead of the uppercase one

## Issues
Closes #269

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
         "@material-ui/icons": "^4.9.1",
         "@material-ui/lab": "^4.0.0-alpha.56",
         "@material-ui/pickers": "^3.3.10",
+        "antalmanac": "file:..",
```
Comment (EricPedley) [inline client/package.json]: Did you add this manually or was it generated by running `npm install`? Not sure what this does or if it's necessary. @LawrenceDavidLu

**Review by EricPedley — COMMENTED** [review]
Looks good. Not sure if it's fine to merge the package / package-lock changes or if we need you to revert those changes. @ChaseC99 what's the play here?

File: client/package.json (hunk 1 above)
Comment (ChaseC99) [reply client/package.json]: Raman and I are discussing this on Discord. Looks like it's from the newer version of npm.

https://discord.com/channels/772739905981644850/780316624020439090/951784755454697532

**Comment by ChaseC99** [comment]
I’m going to upgrade my npm version and then I'll re-review this.

File: client/package.json (hunk 1 above)
Comment (ChaseC99) [reply client/package.json]: Ok so I don't think this should be included.
`package-lock.json` looks fine otherwise.

Comment (ChaseC99) [inline client/package.json]:
```suggestion
```
Please delete this line, run `npm i` again, and then commit/push those changes.

**Review by ChaseC99 — CHANGES_REQUESTED** [review]

**Comment by ChaseC99** [comment]
Also I just wanna point out, the `package-lock.json` stuff is totally not your fault! 

I didn't realize that I was using an older version of npm. Thanks for bringing that to my attention - it should help other fellowship students who open PRs after you.

File: client/package.json (hunk 1 above)
Comment (LawrenceBLu) [reply client/package.json]: Hello! When I delete the line and run `npm install` again, it readds the line. Is that supposed to happen?

Comment (ChaseC99) [reply client/package.json]: What version of npm are you running? 
`npm -v`

Comment (LawrenceBLu) [reply client/package.json]: I am running version 8.5.4.

**Comment by ChaseC99** [comment]
@LawrenceDavidLu can you "Allow edits from maintainers"? 
I'd like to resolve the merge conflicts in the package-lock.


https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

**Comment by LawrenceBLu** [comment]
@ChaseC99 Okay! "Allow edits from maintainers" is now checked.

**Review by ChaseC99 — APPROVED** [review]
Sorry for the confusion around package-lock.json. Thanks so much for your patience and help getting that sorted out! I promise open-source contributions aren't always this confusing lol.

Your change looks solid. Thanks for making that "for" lowercase :)

**Pull request merged**